opentreemap/treemap/js/src/otmTypeahead.js

```diff
@@ -94,7 +94,9 @@ var create = exports.create = function(options) {
             limit: 3000,
             source: function(query, sync, async) {
                 if (query === '') {
-                    sync(prefetchEngine.all());
+                    var result = prefetchEngine.all();
+                    result.sort(sorter);
+                    sync(result);
```

Comment:
Since we now require `sorter`, we need a reasonable default if `sortKeys` isn't passed in.

I think changing
`sorter = _.isArray(options.sortKeys) ? getSortFunction(options.sortKeys) : undefined,`
to
`sorter = _.isArray(options.sortKeys) ? getSortFunction(options.sortKeys) : getSortFunction(['value']),`

would work well.